tests/test_gpu_examples.py

```diff
@@ -658,7 +658,7 @@ def setUp(self):
         from transformers import GPTQConfig
 
         self.causal_lm_model_id = "marcsun13/opt-350m-gptq-4bit"
-        self.quantization_config = GPTQConfig(bits=4, disable_exllama=True)
+        self.quantization_config = GPTQConfig(bits=4, use_exllama=False)
```

Comment:
For exllama v1, i don't think it will be supported in the future as the author moved to exllamav2. I still have to try for exllamav2, so i can leave a todo comment for that.